graphql-schemas/schema.yml

```diff
@@ -1403,7 +1408,8 @@
   - { name: name, type: string, isRequired: true }
   - { name: description, type: string, isRequired: true }
   - { name: onboardingStatus, type: string, isRequired: true }
-  - { name: grafanaUrl, type: string, isRequired: true }
+  - { name: grafanaUrls, isList: true, type: GrafanaDashboardUrls_v1 }
+  - { name: grafanaUrls, isList: true, type: GrafanaDashboardUrls_v1 }
```

Comment:
Duplicated property. I would leave grafanaUrl until you remover ir completely

Reply:
:+1: